gooey/gui/containers/application.py

```diff
@@ -231,6 +234,7 @@ def showSettings(self):
         self.header.setSubtitle(self.buildSpec['program_description'])
         self.footer.showButtons('cancel_button', 'start_button')
         self.footer.progress_bar.Show(False)
+        self.footer.time_remaining_text.Show(False)

```

Comment:
I think you forget to explicitly turn in on for the console screen (or there may be some other bug floating around)? 

If you run a program twice, the timing info doesn't show after the first invocation. e.g.

1. Run [your example](https://github.com/chriskiehl/GooeyExamples/pull/31). 
2. note that the timing info is showing during run 
3. once complete: Click restart / edit + start 
4. No timing information is shown.

Reply:
Amended in `showConsole`.